CodeModel.Tests/Unit/MetricCalculationTests.cs

```diff
@@ -69,6 +69,32 @@ public void Calculates_lack_of_cohesion()
             doctor.Metrics[CaDETMetric.LCOM].ShouldBe(0.75);
         }
 
+        [Fact]
+        public void Calculates_lack_of_cohesion_3()
+        {
+            CodeModelFactory factory = new CodeModelFactory();
+
+            List<CaDETClass> classes = factory.CreateProject(_testDataFactory.GetCohesionClasses()).Classes;
+
+            var dateRange = classes.Find(c => c.Name.Equals("DateRange"));
+            var doctor = classes.Find(c => c.Name.Equals("Doctor"));
+            dateRange.Metrics[CaDETMetric.LCOM3].ShouldBe(0);
+            doctor.Metrics[CaDETMetric.LCOM3].ShouldBe(1.125);
```

Comment:
Did you take this test data and calculated values from the paper? Ideally we would have a test suite that matches the examples provided by the authors.

Reply:
I calculated them manually. Unfortunately, the paper doesn't provide any example.

Reply:
Then look to the web for inspiration.